markdown/events/2022/bytesize-inkscape.md

```diff
@@ -25,3 +25,93 @@ It is our hope that these talks / videos will build an archive of training mater
+That's probably a bad color pairing, sorry, one second, I'll just try this.
+What you can see here is, that the two rings overlap each other. By pressing the page up and the page down, I'm moving what is displaying on top. These buttons are here as well, which does the same thing. You can also move to the very top of the stack so you can have as many objects as you want. For example, I can send this pink one right to the bottom as well, like this.
+
+[11:30](https://www.youtube.com/watch?v=0vKhfedYKGo&t=690)
+Another thing you can do, because everything's based on a coordinate system, is semi-programmatically reorder everything: aligning and distributing them based on some rule. Let's say I select all of these objects and I want to put them all in a single line if I go to `Object` and then `Align and Distribute`, which should open a panel on the right-hand side here. You have various options on how to arrange and distribute your different objects. Let's say I want to have them all in a horizontal line, I can press this button here `Center on Horizontal Axis` and they'll all go into the line here. The same thing also goes for vertical distribution.
+
+[12:08](https://www.youtube.com/watch?v=0vKhfedYKGo&t=728)
+Maybe you want to have everything somewhat equally spaced out. You can see these are closer than these ones, and so on. You can do this with these buttons here on `Distribute`. Then I can blend it, like that, and you can see now, that everything is pretty much ordered in exactly the same way.
+
+[12:29](https://www.youtube.com/watch?v=0vKhfedYKGo&t=749)
+These objects you can also scale and transform. Scaling. Like I said before, you can drag with the arrows here, but you can also go to the `Transform` under the `Object` menu here. You can scale it up programmatically again. I want this one to be twice the size, and press apply, you see twice the size now. If I get the square... You can also rotate things as well, so let's say 45 degree angle, to get a sort of diamond.
+
+[13:00](https://www.youtube.com/watch?v=0vKhfedYKGo&t=780)
+You can also do text and all the text generally is also vector based. You can reorder it, detach them, separate them out and so on. Clicking this text tool here, I can start writing something. The latest version of Inkscape (I've actually accidentally saw the beta version) is a bit slow with the text for some reason. I'm not entirely sure why, so it might be a bit slower than here, but you get the idea. You can make it bigger like this and and you can also color in the same way as the objects as well.
```

Comment:
```suggestion
You can also do text and all the text generally is also vector based. You can reorder it, detach them, separate them out and so on. Clicking this text tool here, I can start writing something. The latest version of Inkscape (I've actually accidentally installed the beta version) is a bit slow with the text for some reason. I'm not entirely sure why, so it might be a bit slower than here, but you get the idea. You can make it bigger like this and and you can also color in the same way as the objects as well.
```